docs/excel/custom-functions-debugging.md

```diff
@@ -44,9 +44,9 @@ You can use VS Code to debug UI-less custom functions in Office Excel on the des
 
 ### Start the VS Code debugger
 
-4. Choose **View > Debug** or enter **Ctrl+Shift+D** to switch to debug view.
-5. From the Debug options, choose **Excel Desktop**.
-6. Select **F5** (or choose **Debug -> Start Debugging** from the menu) to begin debugging. A new Excel workbook will open with your add-in already sideloaded and ready to use.
+4. Choose **View > Run** or enter **Ctrl+Shift+D** to switch to debug view.
+5. From the Run drop-down menu, choose **Excel Desktop (Edge Chromium)**.
+6. Press **F5** (or select **Run -> Start Debugging** from the menu) to begin debugging. A new Excel workbook will open with your add-in already sideloaded and ready to use.
```

Comment:
```suggestion
6. Select **F5** (or select **Run -> Start Debugging** from the menu) to begin debugging. A new Excel workbook will open with your add-in already sideloaded and ready to use.
```
Microsoft Style Guide requires the use of "select" because it is the most accessible language.

Reply:
I see.  Because not all users will be pressing a button?  Makes sense.